### PR TITLE
[DOC-6974] Document Console observability's impact on cluster performance

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-6974] Document Console observability's impact on cluster performance

- Primarily add advice about disabling DB Console metric persistence if you use the Prometheus endpoint with a third-party tool
- Light copyedit
- Done for 23.1 and Cloud. Will backport to 22.2 and 22.1 after technical agreement is reached
- Out of scope: Mention of visus, which will happen in a follow-up PR against the same JIRA.

## Previews
[cockroachcloud/export-metrics.md](https://deploy-preview-16520--cockroachdb-docs.netlify.app/docs/cockroachcloud/export-metrics.html)
[v23.1/datadog.md](https://deploy-preview-16520--cockroachdb-docs.netlify.app/docs/v23.1/datadog.html)
[v23.1/kibana.md](https://deploy-preview-16520--cockroachdb-docs.netlify.app/docs/v23.1/kibana.html)
[v23.1/monitor-cockroachdb-with-prometheus.md](https://deploy-preview-16520--cockroachdb-docs.netlify.app/docs/v23.1/monitor-cockroachdb-with-prometheus.html)
[v23.1/monitoring-and-alerting.md](https://deploy-preview-16520--cockroachdb-docs.netlify.app/docs/v23.1/monitoring-and-alerting.html)
[v23.1/operational-faqs.md](https://deploy-preview-16520--cockroachdb-docs.netlify.app/docs/v23.1/operational-faqs.html)
[v23.1/ui-overview-dashboard.md](https://deploy-preview-16520--cockroachdb-docs.netlify.app/docs/v23.1/ui-overview-dashboard.html)
[v23.1/ui-sessions-page.md](https://deploy-preview-16520--cockroachdb-docs.netlify.app/docs/v23.1/ui-sessions-page.html)
[v23.1/ui-statements-page.md](https://deploy-preview-16520--cockroachdb-docs.netlify.app/docs/v23.1/ui-statements-page.html)
[v23.1/ui-transactions-page.md](https://deploy-preview-16520--cockroachdb-docs.netlify.app/docs/v23.1/ui-transactions-page.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

File: cockroachcloud/export-metrics.md
Comment (mikeCRL) [inline cockroachcloud/export-metrics.md:240]: > The Cloud Console pages that previously showed data about these metrics are still available but visualizations are blank because

I.e. the pages will exist but there will be no data there? If so, could this be slightly clarified?

> no longer persists these metrics

This use of 'persists' seems like dev jargon so I'd propose 'maintains', if accurate.

**Review by mikeCRL — APPROVED** [review]
LGTM except for a couple potential clarifications.

File: cockroachcloud/export-metrics.md
Comment (mikeCRL) [reply cockroachcloud/export-metrics.md]: @mdlinville Just realized that I suggested "maintains" above when I should have suggested "retains"—FWIW! Edited my comment.

File: v23.1/datadog.md
Comment (shannonbradshaw) [inline v23.1/datadog.md:174]: I think this is step 7?

**Review by shannonbradshaw — CHANGES_REQUESTED** [review]

File: v23.1/datadog.md
Comment (mdlinville) [reply v23.1/datadog.md:179]: I also didn't love that second "because". I made an attempt to make this more clear. PTAL.

Comment (shannonbradshaw) [reply v23.1/datadog.md:179]: Thanks.

**Comment by shannonbradshaw** [comment]
LGTM.

File: cockroachcloud/export-metrics.md
Comment (mdlinville) [inline cockroachcloud/export-metrics.md:12]:
```suggestion
To export metrics from a {{ site.data.products.core }} cluster, refer to [Monitoring and Alerting](/docs/{{site.versions["stable"]}}/montoring-and-alerting.html) instead of this page.
```

File: cockroachcloud/export-metrics.md
Comment (kevin-v-ngo) [inline cockroachcloud/export-metrics.md:244]: We should also call out that the DB Console is disabled for Dedicated clusters. 

And @thtruo, can you confirm that the Cloud Console uses the time-series data store as well for Dedicated? If i recall correctly, it uses Prometheus?

Comment (thtruo) [reply cockroachcloud/export-metrics.md:271]: @kevin-v-ngo you're correct - Cloud Console uses Prometheus metrics, a different source from the DB Console

@mdlinville we should should update the copy to just say "disable the DB Console's storage of time-series metrics"

File: v23.1/monitoring-and-alerting.md
Comment (thtruo) [reply v23.1/monitoring-and-alerting.md:31]: +1 that only SQL Activity data is the main request, not metrics generally

The closest thing to this I've heard is customers wanting more granular per-object metrics (like per application, per database, per table, etc) but ack that these don't apply in the context of this doc.

File: cockroachcloud/export-metrics.md
Comment (mdlinville) [reply cockroachcloud/export-metrics.md:271]: I'll revert the addition of this section in the Cloud docs, and mention that DB Console is disabled for Cloud clusters.

File: v23.1/monitoring-and-alerting.md
Comment (mdlinville) [reply v23.1/monitoring-and-alerting.md:31]: @kevin-v-ngo The 10m false assertion came from when I was thinking that the relevant settings were these, rather than the `timeseries` cluster settings. https://www.cockroachlabs.com/docs/stable/cluster-settings.html#setting-sql-stats-flush-interval.

**Comment by mdlinville** [comment]
Reviewers: I addressed all feedback so far and updated the info in the Operational FAQ for consistency. PTAL.

**Comment by mdlinville** [comment]
The latest revision reorganizes the Monitoring and Alerting page a bit more by adding a sub-section each for Metrics Dashboards and SQL Activity pages, adds more context about where SQL Activity data comes from, notes that it is not exported via Prometheus, and updates the UI doc pages for consistency and to add a bit more info.

**Comment by shannonbradshaw** [comment]
@kevin-v-ngo can you hop in and help us get this merged?

File: v23.1/monitoring-and-alerting.md
Comment (kevin-v-ngo) [inline v23.1/monitoring-and-alerting.md:66]: Nice. Thanks for clarifying!

**Review by kevin-v-ngo — APPROVED** [review]
Looks good to me! Thanks for the updates.

**Comment by shannonbradshaw** [comment]
Can we merge this as is and do phase 2 once unblocked?

**Comment by kevin-v-ngo** [comment]
> Can we merge this as is and do phase 2 once unblocked?

Yes - I don't see why we'd block this PR. Phase 2 content would be different focusing on SQL Stats integration.

**Comment by mdlinville** [comment]
Thanks all. @shannonbradshaw I'm assuming your approval to merge.

**Pull request merged**
